qa/rpc-tests/bip68-sequence.py

```diff
@@ -51,15 +51,12 @@ def run_test(self):
         print("Running test BIP68 not consensus before versionbits activation")
         self.test_bip68_not_consensus()
 
-        print("Verifying nVersion=2 transactions aren't standard")
-        self.test_version2_relay(before_activation=True)
+        print("Verifying nVersion=2 transactions are standard")
+        self.test_version2_relay()
```

Comment:
I think we should move this to below the CSV activation.  Testing that relaying of nVersion=2 transactions post activation makes sense, whereas testing that it works pre-activation is strange -- it's okay in this case that we allow it (because on mainnet activation happened a long time ago), but that is not how the code used to behave prior to activation (because relaying those transactions prior to validation opens up some DoS vectors).